src/mbgl/tile/tile_id.hpp

```diff
@@ -126,6 +127,15 @@ inline CanonicalTileID CanonicalTileID::scaledTo(uint8_t targetZ) const {
     }
 }
 
+inline CanonicalTileID CanonicalTileID::operator-(const CanonicalTileID& rhs) const {
```

Comment:
It seems like this definition of subtraction is specific to tile masks. Should it be a named function in `tile_mask.hpp` or `update_tile_masks.hpp` instead?